apps/gp2-server/src/data-providers/user.data-provider.ts

```diff
@@ -198,6 +198,17 @@ export const parseGraphQLUserToDataObject = ({
       return degree;
     });
 
+  const positions =
+    item.positions?.map(({ role, department, institution }) => {
```

Comment:
i was just wondering if a forEach wouldnt be enough? TypeScript might moan but technically it is more fit for the validation that does not actually map anything.